plugins/woocommerce/client/blocks/assets/js/blocks/product-reviews/inner-blocks/review-template/edit.tsx

```diff
@@ -61,58 +58,28 @@ const TEMPLATE = [
 interface ReviewSettings {
 	perPage: number;
 	pageComments: boolean;
-	threadComments: boolean;
-	threadCommentsDepth: number;
 }
 
 const getCommentsPlaceholder = ( {
```

Comment:
I think this can be now simplified to:

```TSX
const getCommentsPlaceholder = ( {
	perPage,
	pageComments,
}: ReviewSettings ): Comment[] => {
	const numberOfComments = pageComments ? Math.min( perPage, 3 ) : 3;

	return Array.from( { length: numberOfComments }, ( _, i ) => ( {
		commentId: -( i + 1 ),
	} ) );
};
```

I'm not sure if it's much clearer, but wanted to mention it just in case, as it took me a while to understand the original code. :sweat_smile:

Reply:
Right!!!